### PR TITLE
Update dockerfile and update ci workflow

### PR DESCRIPTION
**Pull request by schultzp2020**
## Description

Please explain the changes you made here.

Update dockerfile and update ci workflow

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- GitHub Actions are completed and successful
- Unit Tests are updated and passing
- E2E Tests are updated and passing
- Documentation is updated if necessary
- Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,61 +38,57 @@ USER 0
...
-
 # Copy the built packages from the build stage
-COPY --from=build /opt/app-root/src/packages/backend/dist/bundle.tar.gz .
-RUN tar xzf bundle.tar.gz && rm bundle.tar.gz
+COPY --from=build --chown=1001:1001 $REMOTE_SOURCES_DIR/packages/backend/dist/bundle.tar.gz $REMOTE_SOURCES_DIR
+RUN tar xzf $REMOTE_SOURCES_DIR/bundle.tar.gz && rm $REMOTE_SOURCES_DIR/bundle.tar.gz
 
 # Copy any other files that we need at runtime
-COPY ./app-config.yaml ./app-config.production.yaml ./app-config.example.yaml ./app-config.example.production.yaml ./
+COPY --chown=1001:1001 $REMOTE_SOURCES/app-config.yaml $REMOTE_SOURCES/app-config.production.yaml $REMOTE_SOURCES/app-config.example.yaml $REMOTE_SOURCES/app-config.example.production.yaml $REMOTE_SOURCES_DIR
```
Comment (nickboldt) [inline Dockerfile:82]: do we need three versions of app-config in the final container?

**Review by nickboldt — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (christophe-f) [reply Dockerfile:82]: @nickboldt @schultzp2020 No the example files are not needed in the image. This is more to run the application locally.

**Comment by christophe-f** [comment]
@schultzp2020 Did the security issues from the Sonar report resolved?

File: Dockerfile (hunk 1 above)
Comment (schultzp2020) [reply Dockerfile:82]: @christophe-f the example file is required for the helm chart.

**Comment by schultzp2020** [comment]
@christophe-f the issues were false positives.

**Pull request merged**